src/List.mo

```diff
@@ -39,6 +81,15 @@ module {
 
   /// Treat the list as a stack.
   /// This function combines the `head` and (non-failing) `tail` operations into one operation.
+  ///
+  /// Example:
+  /// ```motoko include=initialize
+  /// List.pop<Nat>(?(0, ?(1, null))) // => (?0, ?(1, null))
```

Comment:
```suggestion
  /// List.pop<Nat>(?(0, ?(1, null))) // => [0, 1, null]
```
Seems more readable to me

Reply:
Right, but it doesn't match the output from the interpreter. Is that really better?

ML defines sugar for lists, but that would kick in Motoko whenever something is isomorphic to a list, which might be confusing.